src/linear_algebra/alternating.lean

```diff
@@ -237,3 +237,78 @@ lemma map_congr_perm [fintype ι] (σ : equiv.perm ι) :
 by { rw [g.map_perm, smul_smul], simp }
 
 end alternating_map
+
+open_locale big_operators
+
+namespace multilinear_map
+
+open equiv finset
+
+variables [fintype ι]
+
+private lemma alternization_map_eq_zero_of_eq_aux
+  (m : multilinear_map R (λ i : ι, M) L)
+  (v : ι → M) (i j : ι) (i_ne_j : i ≠ j) (hv : v i = v j) :
+  ∑ (σ : equiv.perm ι), (equiv.perm.sign σ : ℤ) • m.dom_dom_congr σ v = 0 :=
+begin
+  have : ∀ σ, _root_.disjoint {σ} {swap i j * σ},
+  { intros σ,
+    rw [disjoint_singleton, mem_singleton],
+    exact (not_congr swap_mul_eq_iff).mpr i_ne_j },
+
+  apply finset.sum_cancels_of_partition_cancels (perm.mod_swap i j),
+  intros σ _,
+  erw [filter_or, filter_eq', filter_eq', if_pos (mem_univ σ), if_pos (mem_univ (swap i j * σ)),
+    sum_union (this σ), sum_singleton, sum_singleton],
+
+  convert add_right_neg (↑(equiv.perm.sign σ) • m.dom_dom_congr σ v),
+  rw [equiv.perm.sign_mul, equiv.perm.sign_swap i_ne_j, ←neg_smul],
+  rw multilinear_map.dom_dom_congr_apply,
+  rw multilinear_map.dom_dom_congr_apply,
+  congr' 2,
+  { simp },
+  ext, simp [apply_swap_eq_self hv],
+end
```

Comment:
This proof is very similar to

https://github.com/leanprover-community/mathlib/blob/f1b387f8c8f96793f5cea18c1c1441c59aac9925/src/linear_algebra/determinant.lean#L204-L228

I don't know whether it's worth trying to unify them more

Reply:
To me it really depends on how easy to understand the shared part is. I believe something like this would work:
```
lemma sum_eq_zero_of_apply_mul_eq_neg [group G] (x : G) : ∀ f, (∀ σ, f (σ * x) = - f σ) ->
   Σ x : G, f x = 0
```
which specializes to
```
lemma sum_sign_mul_eq_zero_of_apply_mul_eq_self (τ : perm n) : ∀ f, (∀ σ, f (σ * τ) = f σ) ->
   Σ σ, σ.sign * f σ = 0
```
If you agree that that is easy enough to understand, then I would say it's worth the trouble to unify them.

Reply:
Is that first statement true? If it is, I can't use the `mod_swap`-based proof, as `symm` relies on the fact that `x` is its own inverse, and `trans` requires some other stuff...

Reply:
I cleaned up the determinant proof in #5309